Specs/Taylor/0.1.0/Taylor.podspec.json

```diff
@@ -1,29 +1,23 @@
 {
   "name": "Taylor",
   "version": "0.1.0",
-  "summary": "iOS Swift Framework.",
-  "homepage": "https://github.com/nypisces/Taylor",
   "license": "MIT",
+  "summary": "HTTP server written in Swift",
+  "homepage": "https://github.com/izqui/Taylor",
+  "social_media_url": "http://twitter.com/izqui9",
   "authors": {
-    "NY": "nypisces@live.com"
+    "Jorge Izquierdo":"jorge@izqui.me"
   },
   "source": {
-    "git": "https://github.com/nypisces/Taylor.git",
+    "git": "http://twitter.com/izqui9",
```

Comment:
This is clearly incorrect and will not lint. Twitter does not host git at this URI.